scripts/cr_prjcts/cr_user.tcl.in

```diff
@@ -118,41 +118,51 @@ for {set i 0}  {$i < $cfg(n_config)} {incr i} {
         if {$cfg(load_apps) eq 1} {
             set tmp_val "[format "\$vfpga_c$i\_$j"]" 
             set tmp_path "[format "%s" $tmp_val]"
-            set cmd "[format "set tmp_path $tmp_path"]"
+            set cmd "[format "set tmp_path \"$tmp_path\""]"
             eval $cmd
+            set src_dirs [split $tmp_path]
+            set vfpga_src_dir [lindex $src_dirs 0]
 
             # Add all raw HDL files (no packaging)
             file mkdir "$proj_dir/hdl/ext/raw"
             set files_hdl [glob -nocomplain "$hw_dir/hdl/pkg/axi_macros.svh" "$hw_dir/hdl/pkg/axi_intf.sv" "$hw_dir/hdl/pkg/lynx_macros.svh" "$hw_dir/hdl/pkg/lynx_intf.sv"]
             file copy -force {*}$files_hdl "$proj_dir/hdl/ext/raw"
 
-            if {[file isdirectory "${CMAKE_SOURCE_DIR}/$tmp_path/hdl"]} {
-                set files_hdl [glob -nocomplain "${CMAKE_SOURCE_DIR}/$tmp_path/hdl/*"]
-                if {[llength $files_hdl]} {
-                    file copy -force {*}$files_hdl "$proj_dir/hdl/ext/raw"
+            for {set i 0} {$i < [llength $src_dirs]} {incr i} {
```

Comment:
Won't this be a problem with multi-tenancy? Isn't there already a variable i at the top of the file, that iterates over vFPGA.

A cleaner solution would be: foreach src_dir $src_dirs, no?